### ISSUE TITLE
Custom URI schemes never load and don't call callback

### ISSUE THREAD

**Issue by jeandrek** [Jonathan50/kea-browser#3]
?

**Comment by jeandrek** [comment]
``` c
void
register_schemes(WebKitWebContext *context)
{
  webkit_web_context_register_uri_scheme(context, "about",
                        (WebKitURISchemeRequestCallback)about_protocol_request,
                        NULL, NULL);
}
```

it should work... I can't see anything wrong with it...

**Comment by jeandrek** [comment]
I'll try

``` c
void
register_schemes(WebKitWebContext *context)
{
  webkit_web_context_register_uri_scheme(context, "about",
                        WEBKIT_URI_SCHEME_REQUEST_CALLBACK(about_protocol_request),
                        NULL, NULL);
}
```

but it's odd I get no error...

**Comment by jeandrek** [comment]
NOO IT DOESN'T WORK IT'S NOT DEFINED

**Comment by jeandrek** [comment]
https://bugs.webkit.org/show_bug.cgi?id=124405
srsly? It doesn't explain what was wrong??

**Comment by jeandrek** [comment]
Argh
no
why

**Comment by jeandrek** [comment]
halp

**Comment by jeandrek** [comment]
hmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm

**Comment by serverhiccups** [comment]
The sooner this gets fixed the sooner we are to v0.2S1...

**Comment by serverhiccups** [comment]
That bug was never a bug.

**Comment by ghost** [comment]
Hi, I'm confused by this too = =
You use `kea-about` as scheme, and may do some uri rewrititg like `ephy_embed_utils_normalize_address`.

**Comment by jeandrek** [comment]
@ghost Wait. You're right. If I make it register something different from just `about`, it works!

**Comment by jeandrek** [comment]
This is mysterious. Thanks.

**Comment by jeandrek** [comment]
After almost two years, it is fixed with v0.2.5!!!!!!!!!
